CRM/Contact/Form/DedupeFind.php

```diff
@@ -51,6 +51,7 @@ public function buildQuickForm() {
     $groupList = array('' => ts('- All Contacts -')) + CRM_Core_PseudoConstant::nestedGroup();
 
     $this->add('select', 'group_id', ts('Select Group'), $groupList, FALSE, array('class' => 'crm-select2 huge'));
+    $this->add('text', 'limit', ts('No of contacts to find matches for '));
```

Comment:
Ah it seems to be a caching thing. If you search for 1000 and then for 20 the results don't change. Pesky.